packages/frontend/src/components/UnderlyingData/UnderlyingDataModalContent.tsx

```diff
@@ -135,15 +136,31 @@ const UnderlyingDataModalContent: FC<Props> = () => {
               ]
             : [];
 
+        // Metric filters fieldId don't have table prefixes, we add it here
+        const metric: Metric | undefined =
+            isField(meta?.item) && isMetric(meta.item) ? meta.item : undefined;
+        const metricFilters =
+            metric?.filters?.map((filter) => {
+                return {
+                    ...filter,
+                    target: {
+                        fieldId: getFieldId({
```

Comment:
I'm doing this here instead of doing it on the translator because I imagine in the future we might want to filter on other tables, in that case it is easier to do it here than in the translator.

Reply:
These should be used to generate the following sql (sorry this wasn't in the ticket):

```
metrics:
  total_active_users:
    type: count
    filters:
      - is_active: true
    sql: ${TABLE}.user_id
```

```sql
CASE WHEN is_active = true THEN ${TABLE}.user_id ELSE NULL END
```

Reply:
So no filters are added to the table, it's baked into the generated sql for the metric

Reply:
It should also be here though, so that when you drill into a metric, you should only see the relevant rows used to calculate the metric

Reply:
But isn't our current filter for underlying data already doing that ? is going to show the rows for that metric value.